prisma/loadCsv.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-console */
+import { parse } from 'csv-parse/sync';
+import * as fs from 'fs';
+
+// loads a CSV file from the filesystem, ready for parsing
+export default function loadCSVFromFile(filename: string, discardHeader = true, delimiter = ',') {
```

Comment:
nit: could make this an async function. not a big deal since it's just for our seed data.

Reply:
🤔 I took a look, but which contained methods are async? `readFileSync` gives me a "this has no effect" And the parse functions are all synchronous

Reply:
I would switch `readFileSync` to `fs.promises.readFile`. 
Generally, my recommendation is to `await`/ promisify any external dependency or slow, complex function 🔢